vdom/src/com/vdom/api/Card.java

```diff
@@ -113,5 +114,7 @@
 
     public PileCreator getPileCreator();
 
+    public Type[] getTypes();
```

Comment:
This method seems unnecessary. I'd be in favor of removing it for now unless there is a need.